src/v2/Components/AuctionBuyersPremiumDialog.tsx

```diff
@@ -79,12 +79,16 @@ const AuctionBuyersPremiumDialog: React.FC<AuctionBuyersPremiumDialogProps> = ({
             </Text>
           ) : (
             schedule.map((premium, i) => {
+              const percent = (premium.percent ?? 0) * 100
```

Comment:
non-blocking: if it's something we use shared across other components maybe putting in some utils folder

Reply:
So far its only used here